core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -924,6 +921,10 @@ public ServerBuilder serverListener(ServerListener serverListener) {
         return this;
     }
 
+    public int getMaxNumConnections() {
+        return maxNumConnections;
```

Comment:
We don't usually provide a getter in a builder. If you are exposing it for testing, please make it package-private and add `@VisibleForTesting`. Also, please rename the method to `maxNumConnections()` because we do not prepend `get` for the getter methods mostly.